flytekit/clis/sdk_in_container/register.py

```diff
@@ -148,14 +164,30 @@ def register(
     """
     see help
     """
+    if copy is not None and non_fast:
+        raise ValueError("--non-fast and --copy cannot be used together. Use --copy none instead.")
+
+    # Error handling for non-fast/copy conflicts
+    if copy == CopyFileDetection.NO_COPY:
+        non_fast = True
+        # Set this to None because downstream logic currently detects None to mean old logic.
+        copy = None
+    elif copy == CopyFileDetection.ALL:
+        if non_fast:
+            raise ValueError("Conflicting options: cannot specify both --non-fast and --copy all")
+    elif copy == CopyFileDetection.LOADED_MODULES:
+        if non_fast:
+            raise ValueError("Conflicting options: cannot specify both --non-fast and --copy auto")
+    ls_files = ctx.obj[constants.CTX_VERBOSE] > 0
```

Comment:
Can we be explicit and rename `ls_files` to "show_files" everywhere?

```suggestion
    show_files = ctx.obj[constants.CTX_VERBOSE] > 0
```

Reply:
i want to keep the function name, but these variables yeah sure.